test/controllers/unattended_controller_test.rb

```diff
@@ -57,6 +57,23 @@ class UnattendedControllerTest < ActionController::TestCase
     assert_response :success
   end
 
+  test "should get a kickstart if MAC is provided with two hosts with same MAC" do
+    host1 = FactoryBot.create(:host, :managed, :with_dhcp_orchestration, :build => true,
+      :name => "host2_same_mac",
+      :mac => @rh_host.mac,
+      :operatingsystem => operatingsystems(:ubuntu1010),
+      :ptable => @ptable_ubuntu,
+      :medium => media(:ubuntu),
+      :architecture => architectures(:x86_64),
+      :organization => @org,
+      :location => @loc)
+    get :host_template, params: { :kind => 'finish', :mac => host1.mac }
+    assert @rh_host.created_at < host1.created_at
```

Comment:
Interesting, this assert failed on jenkins but only for MySQL and just once (one Ruby version). Why would `rh_host` not be created before `host1`?